contracts/protocol/facets/SequenatialCommitHandlerFacet.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.9;
+
+import { IBosonSequentialCommitHandler } from "../../interfaces/handlers/IBosonSequentialCommitHandler.sol";
+import { IBosonVoucher } from "../../interfaces/clients/IBosonVoucher.sol";
+import { DiamondLib } from "../../diamond/DiamondLib.sol";
+import { PriceDiscoveryBase } from "../bases/PriceDiscoveryBase.sol";
+import { ProtocolLib } from "../libs/ProtocolLib.sol";
+import { FundsLib } from "../libs/FundsLib.sol";
+import "../../domain/BosonConstants.sol";
+import { Address } from "../../ext_libs/Address.sol";
+import { Math } from "../../ext_libs/Math.sol";
+
+interface WETH9Like {
+    function withdraw(uint256) external;
+
+    function deposit() external payable;
+
+    function transfer(address, uint256) external returns (bool);
+
+    function transferFrom(
+        address,
+        address,
+        uint256
+    ) external returns (bool);
+}
```

Comment:
please remove this and use the file instead

Reply:
Removed. Wasn't even used anywhere in this file.